### DISCUSSION TITLE
Feature Request: Update entity

### DISCUSSION THREAD

**Discussion by milandzuris** [dougiteixeira/proxmoxve#131] [Feature request] [answered]
HI, Please make a Update Button for Udaptes, or this Button only for whats the Actual Version or Actual Version and Installed Version

BIG FEATURE for Everyone

**Comment by dougiteixeira** [comment]
I don't really like the idea of ​​running the update command from the Home Assistant as it might impact the Proxmox server and VM/Containers. But I will consider the suggestion to include the update entity in the future to display the latest version and installed version.

**Comment by Gugulator** [comment]
is it possible to add binary sensor that shows if system restart is pending?
need to generate rsa key:
`ssh-keygen -t rsa`
then copy it to proxmox:
`ssh-copy-id user@proxmox_ip`
and final ssh command will looks like:
`"ssh -t user@proxmox_ip '[ -e /run/reboot-required ] && echo on || echo off'"`

**Reply by dougiteixeira** [reply, marked as answer]
I have included a binary sensor that will be active when updates are available and a sensor with the total number of updates and package list as attribute (#158) and will be available in the next release.

I am not in favor of adding the action of updating packages via Home Assistant, as this could impact the server's operability, the integration cannot assume this risk.

**Reply by Gugulator** [reply]
wow! it is perfect solution! thank you so much!
